server.js

```diff
@@ -1,27 +1,11 @@
-app.get('/slambook', async (req, res) => {
+app.post('/slambook', async (req, res) => {
   try {
     

```

Comment:
Using try-catch blocks would be beneficial to handle any potential errors that may occur when saving the new entry to the database.